packages/features/src/Domain/Feature/Feature.ts

```diff
@@ -28,5 +28,5 @@ export type Feature = {
     borderColor: string;
   };
   deletionWarning?: string;
-  expiresAt?: number;
+  expiresAt?: Date;
```

Comment:
hmm this should be a number in microseconds - I think everywhere where it's possible we should microsecond timestamps instead of Dates, wdyt?

Reply:
Yeah I don't recall my reasoning behind this change 😅 will revert it